.grit/patterns/python/py_no_debugger.md

```diff
@@ -10,15 +10,61 @@ tags: #fix #good-practice
 engine marzano(0.1)
 language python
 
-`` as $body where {
-    and {
-        $body <: contains `import pdb` => .,
-        $body <: contains `pdb.set_trace()` => .
+file($name, $body) where {
+     or {
+        and {
```

Comment:
Do we really need *all* these conditions to be true? I doubt it. I think you want to individually remove *any* of these if found.

Reply:
Hi @morgante, you was correct, updated it with `any`.

don't know why `macos` does not support `any` keyword, I got this error in my local setup also.